database/sdc-seed-json.js

```diff
@@ -0,0 +1,51 @@
+const fs = require ('fs');
+const file = fs.createWriteStream('./seedData.json');
+
+var randomNumberGenerator = () => {
+  var randomNumber = 1000 + Math.floor(Math.random() * 1000);
+  return randomNumber;
+}
+
+var createData = (productNumber, randomNumber) => {
+	var i = 0;
```

Comment:
Mmh again with the mixed syntax for no particular reason.

You're swapping between var and let and I can't tell if there's a reason why. Also, between while and for loops.

Either is ok, but mixing them implies that there's a benefit to one over the other in this use case, or again, sloppy